### PR TITLE
Create 2020-11-16-mounts.md

### PR DESCRIPTION
**Pull request by YurkoWasHere**

### CODE REVIEW

File: operations/hardware-budgets/2020-11-16-mounts.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+_The sign-off should include at least one experienced member in the Network working group._
+
+Members of Network Planning, Design and Operations:
+- @yurkowashere
+- @name
+
+Grant Administrators:
+- @YurkoWasHere 
+- @name
```
Comment (benhylau) [inline operations/hardware-budgets/2020-11-16-mounts.md:42]:
```suggestion
- @benhylau
```

**Review by benhylau — APPROVED** [review]

File: operations/hardware-budgets/2020-11-16-mounts.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# 2020-11-16 Hardware Budget for Toronto Community Network
+
+## Purpose
+
+_What is the hardware for? If this is related to a particular deployment and require approval by a specific date, please mention it here._
+
+Purhcase 5 mounts to have in stock and deply as needed.
```
Comment (benhylau) [inline operations/hardware-budgets/2020-11-16-mounts.md:7]:
```suggestion
Purchase 5 mounts to have in stock and deploy as needed.
```

File: operations/hardware-budgets/2020-11-16-mounts.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+## Hardware List
+
+_A list of all hardware items and the prices from specific vendors we plan to order from._
+
+| Item        | Vendor     | Price  | Currency | Quantity | 
+|:------------|:-----------|-------:|---------:|---------:|
+| CS-18274(1)    | Infinity Cables|  180   |   CAD    |        5 |
+| Cinder Blcks| Home Depot |    5   |   CAD    |        20 |
+
+(1) CS-18274: Custom Southwinds Engineering Roof Mount as per Cisco specs FOB Flesherton, Ontario facility.
```
Comment (benhylau) [inline operations/hardware-budgets/2020-11-16-mounts.md:18]:
```suggestion
(1) CS-18274: Custom Southwinds Engineering Roof Mount as per Cisco specs for the Digital Canopy project. FOB Flesherton, Ontario facility.
```

**Comment by ultush** [comment]
No changes from me

**Review by ultush — APPROVED** [review]

**Review by Pedro-on-a-bike — APPROVED** [review]

**Review by TimTor — APPROVED** [review]

**Pull request merged**